website/content/docs/security/acl/acl-policies.mdx

```diff
@@ -9,6 +9,8 @@ description: >-
 
 This topic describes policies, which are components in Consul's access control list (ACL) system. Policies define which services and agents are authorized to interact with resources in the network.
 
+Looking for an policy example for your use case? Refer to the [Policy Examples](/consul/docs/security/acl/acl-policy-examples) topic.
```

Comment:
```suggestion
Refer to the [Policy Examples](/consul/docs/security/acl/acl-policy-examples) for example policy configurations.
```
Rhetorical and call-and-response type questions are prohibited per the style guide.